src/styles/modal.css

```diff
@@ -0,0 +1,83 @@
+/* modal */
+
+.ui.modal {
+  background-color: var(--shade-white);
+  border-radius: 0;
+  box-shadow: var(--box-shadow-5);
+}
+
+.ui.modal > .icon:first-child + * {
+  border-radius: 0;
+}
+
+/* close icon */
+
+.ui.modal > .close {
+  color: var(--shade-white);
+  font-size: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+  right: var(--spacing-2);
+  top: var(--spacing-3);
+  transition: opacity var(--transition-hover);
+  width: 1.5rem;
+}
+
+@media only screen and (max-width: 991px) {
+  .ui.modal > .close {
+    color: var(--shade-white);
+    font-size: 1.5rem;
+    height: 1.5rem;
+    padding: 0;
+    right: var(--spacing-2);
+    top: var(--spacing-3);
+    transition: opacity var(--transition-hover);
+    width: 1.5rem;
+  }
+}
+
+/* title */
+
+.ui.modal > .header {
```

Comment:
I believe that the closing icon should be aligned `center` with the header text:

![image](https://github.com/abby-moffitt/GA-Semantic/assets/101662434/13501cc2-468e-4a76-8826-26339081f835)

Reply:
But for smaller screens, it is not right now:

<img width="561" alt="image" src="https://github.com/abby-moffitt/GA-Semantic/assets/101662434/f6d584c6-33ee-4d7b-b3a5-828999ed097b">